NEWS.md

```diff
@@ -1,4 +1,36 @@
-# mrgsolve (development version)
+# mrgsolve 1.5.3
+
+- Added `END_OF_INFUSION` syntax, which is `true` on the internal record that 
+  stops an infusion (#1288). 
+
+- `mwrite_*()` now render block names in upper case, even for blocks coded
```

Comment:
Okay as is, but explicitly writing `mwrite_cpp()` and `mwrite_yaml()` would be good to enable pkgdown to auto-link to the docs.

Reply:
Thanks; updated.